apps/test/unit/code-studio/pd/professional_learning_landing/LandingPageWorkshopsTableTest.jsx

```diff
@@ -143,18 +139,21 @@ describe('EnrolledWorkshops', () => {
     expect(preWorkshopSurveyButton.props().disabled).to.be.true;
   });
 
-  it('shows Status column and only has Workshop Details button when forMyPlPage is true', function () {
-    const enrolledWorkshopsTable = shallow(
-      <WorkshopsTable workshops={workshops} forMyPlPage={true} />
+  it('shows Status column and only has Workshop Details button when not in a participant view', function () {
+    const workshopsTable = shallow(
+      <LandingPageWorkshopsTable
+        workshops={workshops}
+        participantView={false}
```

Comment:
Since other uses of `LandingPageWorkshopsTable` above set `participantView` as falsey by not including it at all, could the test reflect that too by not having the `participantView` parameter specified? I know it should theoretically work either way but I think I get wary around testing `null` vs. `false` when sometimes they cause different issues.

Reply:
actually, on second thought, I think I'd rather be explicit in `LandingPage`

Reply:
Works for me!